Guide/mail.markdown

```diff
@@ -107,13 +130,14 @@ config = do
     option $ SMTP
         { host = "smtp.myisp.com"
         , port = 2525
-        , credentials = Nothing -- or Just ("myusername","hunter2")
+        , credentials = Nothing -- or: Just ("myusername","hunter2")
+		, encryption = TLS -- <-- other options: `Unencrypted` or `STARTTLS`
```

Comment:
Something is off with the indentation here and below

Reply:
Opps, sorry, missed some leftover tabs in there. Thanks for letting me know!